lib/protocol/bep.proto

```diff
@@ -100,6 +100,7 @@ message FileInfo {
     uint32       permissions    = 4;
     int64        modified_s     = 5;
     int32        modified_ns    = 11;
+    uint64       modified_by    = 12 [(gogoproto.customtype) = "ShortID", (gogoproto.nullable) = false];
```

Comment:
Eh... Yeah. But it's not live on the web. Which is not your fault. I'll go fix it, sorry for the noise.